tests/integration/mounts.bats

```diff
@@ -52,3 +57,28 @@ function teardown() {
 	runc run test_busybox
 	[ "$status" -eq 0 ]
 }
+
+@test "runc run [rw bind mount of a ro fuse sshfs mount]" {
```

Comment:
Can we move this into `mounts_sshfs.bats`, as this has huge extra depenedncy?

Or maybe just skip the test by default and require some env var to opt-in to this test.

Reply:
Will do.

Reply:
Done. If sshfs is not installed or not working for some reason, the test is skipped.